src/tools/admin/conseillers/change-date-naissance.js

```diff
@@ -0,0 +1,47 @@
+const { program } = require('commander');
+const { execute } = require('../../utils');
+const dayjs = require('dayjs');
+
+const formatDate = date => {
+  return dayjs(date, 'YYYY-MM-DD').toDate();
+};
+
+execute(__filename, async ({ db, logger, Sentry, exit }) => {
+
+  program.option('-d, --date <type>', 'date : date de naissance sous le format AAAA/MM/DD');
+  program.option('-i, --id <id>', 'id: id PG du conseiller');
+  program.helpOption('-e', 'HELP command');
+  program.parse(process.argv);
+
+  let id = ~~program.id;
```

Comment:
pourquoi tu as utilisé `~~` ici ? si c'est pour supprimer les chiffres après la virgule, tu peux faire :
```suggestion
  const id = Math.floor(program.id);
```

Reply:
en faite sans ce symbole, l'id sera seulement de type string, et il faut que ça sois de type int ! mais c'est vrai qu'il y a plusieurs manière de le faire